test/common/core/coretype.spec.js

```diff
@@ -1161,11 +1161,30 @@ describe('coretype', function () {
                     paths.push(core.getPath(instances[0]));
                     paths.push(core.getPath(instances[1]));
                     expect(paths).to.have.members(['/template', '/template/child']);
-                    if(--neededChecks === 0){
+                    if (--neededChecks === 0) {
                         done();
                     }
                 }, core.loadInstances(core.getBase(newChild)));
             }, core.loadByPath(newRoot, '/template/child'));
         }, core.loadRoot(core.getHash(root)));
     });
+
+    it('should remove atr and reg field of an instance during persist', function (done) {
```

Comment:
probably a dumb question but these can be removed, right? Are the attributes and registry values stored in different fields?